test/liquidityConversionRate.js

```diff
@@ -531,19 +531,19 @@ contract('kyberReserve for Liquidity', function(accounts) {
                 feeInBps,
                 maxSellRateInPrecision,
                 minSellRateInPrecision
-            ) 
+            )
     });
 
     it("should init reserve and set all reserve data including balances", async function () {
-        reserveInst = await Reserve.new(network, liquidityConvRatesInst.address, admin);
-        await reserveInst.setContracts(network, liquidityConvRatesInst.address, 0);
+        reserveInst = await Reserve.new(network, liqConvRatesInst.address, admin);
```

Comment:
don't follow how these changes got in this PR

Reply:
Was causing the test to fail because it was referencing the wrong conversion rates contract.